readthedocs/audit/templates/audit/list_logs.html

```diff
@@ -10,79 +10,105 @@
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">downloaded a document</a>
             {% endblocktrans %}
-          {% endif %}
-        {% elif log.action == AuditLog.DOWNLOAD %}
-          {% if log.log_user_id %}
-            {% blocktrans trimmed with action=log.action user=log.log_user_username path=log.resource %}
-              <a href="?user={{ user }}">
-                <code>{{ user }}</code>
-              </a>
-              <a href="?action={{ action }}" title="{{ path }}">downloaded</a> a document
+          {% elif log.action == AuditLog.INVITATION_SENT %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">sent an invitation</a>
             {% endblocktrans %}
-          {% else %}
-            {% blocktrans trimmed with action=log.action user=log.log_user_username path=log.resource %}
-              A user <a href="?action={{ action }}" title="{{ path }}">downloaded</a> a document
+          {% elif log.action == AuditLog.INVITATION_REVOKED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">revoked an invitation</a>
+            {% endblocktrans %}
+          {% elif log.action == AuditLog.INVITATION_ACCEPTED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">accepted an invitation</a>
+            {% endblocktrans %}
+          {% elif log.action == AuditLog.INVITATION_DECLINED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">declined an invitation</a>
             {% endblocktrans %}
           {% endif %}
-        {% endif %}
-
-        {% trans "from" %}
-
-        <a href="?ip={{ log.ip }}" title="{{ log.browser }}">
-          <code>{{ log.ip }}</code>
-        </a>
-
-        {# If the authentication was on a doc domain. #}
-        {% if log.log_project_id %}
-          {% trans "on" %}
-          {% if log.project %}
-            <a href="{% url 'projects_detail' log.log_project_slug %}">
-              <code>{{ log.log_project_slug }}</code>
-            </a>
-          {% else %}
-            {# The original project has been deleted, don't link to it. #}
-            <code title="{{ log.resource }}">{{ log.log_project_slug }}</code>
-          {% endif %}
-        {% endif %}
+        </div>
 
         <span class="quiet right" title="{{ log.created|date:"DATETIME_FORMAT" }}">
           {% blocktrans trimmed with log.created|timesince as date %}
             {{ date }} ago
           {% endblocktrans %}
         </span>
+
+        {# Data about the log entry #}
```

Comment:
GitHub displays logs as

![Screenshot 2022-09-20 at 13-36-35 Security log](https://user-images.githubusercontent.com/4975310/191337900-480935dc-1772-4fde-b9be-80369220a665.png)

They show the IP and location

Reply:
This matches what I was describing the other day. The main translated string is "Country changed from your previous session", without introducing variables, and metadata is listed below.